File: C++/2042. Check if Numbers Are Ascending in a Sentence.cpp
class Solution {
public:
    bool areNumbersAscending(string s) {
        bool t=false;
        string k="";
        int a=0;
        for(char c:s){
            if(t&&c==' '){
                if(stoi(k)<=a)
                return false;
                else
                a=stoi(k);
                k="";
                t=false;
            }
            if(c<='9'&&c>='0'){
                t=true;
                k+=c;
            }
        }
        if(t){
            if(stoi(k)<=a)
            return false;
        }
        return true;
    }
};

File: JAVA/2042. Check if Numbers Are Ascending in a Sentence.java
class Solution {
    public boolean areNumbersAscending(String s) {
        boolean t=false;
        String k="";
        int a=0;
        for(int i=0;i<s.length();i++){
            if(t==true&&s.charAt(i)==' '){
                if(Integer.parseInt(k)<=a)
                return false;
                else
                a=Integer.parseInt(k);
                k="";
                t=false;
            }
            if(s.charAt(i)<='9'&&s.charAt(i)>='0'){
                t=true;
                k+=s.charAt(i);
            }
        }
        if(t==true){
            if(Integer.parseInt(k)<=a)
                return false;
                else
                a=Integer.parseInt(k);
        }
        return true;
    }
}